spec/relativetimeformat.html

```diff
@@ -1,61 +1,256 @@
         1. Else
           1. Let _options_ be ? ToObject(_options_).
         1. Let _opt_ be a new Record.
         1. Let _matcher_ be ? GetOption(_options_, *"localeMatcher"*, *"string"*, «*"lookup"*, *"best fit"*», *"best fit"*).
-        1. Set _opt_.[[localeMatcher]] to _matcher_.
-        1. Let _localeData_ be *%RelativeTimeFormat%*.[[localeData]].
-        1. Let _r_ be ResolveLocale(*%RelativeTimeFormat%*.[[availableLocales]], _requestedLocales_, _opt_, *%RelativeTimeFormat%*.[[relevantExtensionKeys]], _localeData_).
-        1. Set _relativeTimeFormat_.[[locale]] to the value of _r_.[[locale]].
-        1. Let _dataLocale_ be _r_.[[dataLocale]].
+        1. Set _opt_.[[LocaleMatcher]] to _matcher_.
+        1. Let _localeData_ be *%RelativeTimeFormat%*.[[LocaleData]].
+        1. Let _r_ be ResolveLocale(*%RelativeTimeFormat%*.[[AvailableLocales]], _requestedLocales_, _opt_, *%RelativeTimeFormat%*.[[RelevantExtensionKeys]], _localeData_).
+        1. Set _relativeTimeFormat_.[[Locale]] to the value of _r_.[[Locale]].
+        1. Let _dataLocale_ be _r_.[[DataLocale]].
         1. Let _s_ be ? GetOption(_options_, *"style"*, *"string"*, «*"long"*, *"short"*, *"narrow"*», *"long"*).
-        1. Set _relativeTimeFormat_.[[style]] to _s_.
+        1. Set _relativeTimeFormat_.[[Style]] to _s_.
         1. Let _u_ be ? GetOption(_options_, *"unit"*, *"string"*, «*"best fit"*, *"second"*, *"minute"*, *"hour"*, *"day"*, *"week"*, *"month"*, *"quarter"*, *"year"*», *"best fit"*).
-        1. Set _relativeTimeFormat_.[[unit]] to _u_.
+        1. Set _relativeTimeFormat_.[[Unit]] to _u_.
         1. Let _dataLocaleData_ be Get(_localeData_, _dataLocale_).
         1. Let _fields_ be Get(_dataLocaleData_, *"fields"*).
         1. Assert: _fields_ is an object (see <emu-xref href="#sec-Intl.RelativeTimeFormat-internal-slots"></emu-xref>).
-        1. Set _relativeTimeFormat_.[[fields]] to _fields_.
-        1. Let _pluralRules_ be Get(_dataLocaleData_, *"pluralRules"*).
-        1. Assert: _pluralRules_ is an object (see <emu-xref href="#sec-Intl.RelativeTimeFormat-internal-slots"></emu-xref>).
-        1. Set _relativeTimeFormat_.[[pluralRule]] to Get(_pluralRules_, *"cardinal"*).
-        1. Let _nf_ be Construct(%NumberFormat%, «[_r_.[[locale]], {minimumIntegerDigits: 2, useGrouping: *false*}»).
-        1. Set _relativeTimeFormat_.[[numberFormat]] to _nf_.format.
-        1. Set _relativeTimeFormat_.[[boundResolve]] to *undefined*.
-        1. Set _relativeTimeFormat_.[[initializedRelativeTimeFormat]] to *true*.
+        1. Set _relativeTimeFormat_.[[Fields]] to _fields_.
+        1. Let _nf_ be Construct(%NumberFormat%, «[_r_.[[Locale]], {minimumIntegerDigits: 2, useGrouping: *false*}»).
+        1. Set _relativeTimeFormat_.[[NumberFormat]] to _nf_.format.
+        1. Let _pr_ be Construct(%PluralRules%, «[_r_.[[Locale]], {style: *"cardinal"*}»).
+        1. Set _relativeTimeFormat_.[[PluralRules]] to _pr_.select.
```

Comment:
you could still do `GetMethod()`, and bind it to _pr_. But I will argue that we should just store the instance, I'm not sure about this. Additionally, we have access to those abstract operations directly, we could hold the instance and call the abstract operation directly when needed instead of holding to the public method.